spec/reflexes/products_reflex_spec.rb

```diff
@@ -32,4 +32,88 @@
       end
     end
   end
+
+  describe '#bulk_update' do
+    let!(:product_b) { create(:simple_product, name: "Bananas") }
+    let!(:product_a) { create(:simple_product, name: "Apples") }
+
+    it "saves valid changes" do
+      params = {
+        # '[products][<i>][name]'
+        "products" => [
+          {
+            "id" => product_a.id.to_s,
+            "name" => "Pommes",
+          }
+        ]
+      }
+
+      expect{
+        run_reflex(:bulk_update, params:)
+        product_a.reload
+      }.to change(product_a, :name).to("Pommes")
```

Comment:
This is an old syntax for `change`. I think it's deprecated.
```suggestion
      }.to change { product_a.name }.to("Pommes")
```

Reply:
I think your suggestion is more readable (and functionally equivalent), so I'll accept that change.

The documentation seems to prefer the other receiver/message format though: 
https://rubydoc.info/gems/rspec-expectations/RSpec%2FMatchers:change

I'm not sure, but would presume that the receiver/message format results in more tailored error messages. But honestly I'd agree it's unnecessary. 

So would you agree to set the linter to prefer the block format? 
https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ExpectChange  `EnforcedStyle: block`

Reply:
Yes, I would agree to enforce the block style because it's more flexible. For some cases you need the block style and having two different styles floating around is confusing.

I also find that the block style is more clear. Ruby developers know blocks but when you pass an object and a symbol to the `change` method, it's not obvious what it does under the hood. If I didn't know the matcher and saw `change(product, :name)` then I would wonder: does this only work on ActiveRecord attributes? Does it check the database or the parameters? Can I test private methods this way?